Apps/WebClient/src/ClientApp/src/views/publicCovidTest.vue

```diff
@@ -470,6 +470,8 @@ export default class PublicCovidTestView extends Vue {
                                 {{
                                     publicCovidTestResponseResultError.description
                                 }}
+                                <br />
+                                {{ publicCovidTestResponseResultError.detail }}
```

Comment:
Nit: I'd suggest wrapping the `description` in a `<p>` tag and the `detail` in a `<div>` for better spacing between the paragraphs.